GameServer/packets/Client/168/DetailDisplayHandler.cs

```diff
@@ -2129,8 +2131,16 @@ public static string DelveStyle(GameClient clt, int id)
 				//.Value("Skill",43)
 				if (style.GrowthRate>0)
 					dw.AddKeyValuePair("OpeningDamage",style.GrowthRate*100);
-				//.Value("SpecialValue", GetSpecialValue(style),GetSpecialValue(style)!=0)
-				//.Value("FollowupStyle",style.DelveFollowUpStyles,!string.IsNullOrEmpty(style.DelveFollowUpStyles))
+                //.Value("SpecialValue", GetSpecialValue(style),GetSpecialValue(style)!=0)
+                //.Value("FollowupStyle",style.DelveFollowUpStyles,!string.IsNullOrEmpty(style.DelveFollowUpStyles))
+                if (style.Procs != null && style.Procs.Count > 0)
+                {
+                    foreach (Tuple<Spell, int, int> proc in style.Procs)
+                    {
+                        dw.AddKeyValuePair("SpecialNumber", proc.Item1.InternalID);
+                        dw.AddKeyValuePair("SpecialType", 1);

```

Comment:
`SpecialType` value depends on the spell which is actually put in the proc.
For this reason I believe having a `TooltipDelveStyle` override in `SpellHandler` for various spells would make sense as each of the spells has its own way to delve itself when put in a style.